### PR TITLE
Apply copy improvements & Electrum UX cleanup

### PR DESCRIPTION
**Pull request by otkstdio**
This improves the electrum connection UX and other copy changes throughout the app

### CODE REVIEW

File: src/cryptoadvance/specter/templates/includes/sidebar/sidebar.jinja [hunk 1]
```diff
@@ -16,9 +16,7 @@
...
...8,-0.0790377 -0.529348,-0.219322l-6.13412,-6.13622l-1.41661e-07,-1.41718e-07c-0.374915,-0.375067 -0.585519,-0.883682 -0.5855,-1.414v-1.5051l2.13163e-14,3.01992e-07c-1.66785e-07,-1.10457 0.89543,-2 2,-2Zm0,44h-9h-8.74228e-08c-1.10457,-4.82823e-08 -2,-0.895431 -2,-2c0,0 0,-1.42109e-14 0,-1.42109e-14v-12v0c0,-1.10457 -0.895431,-2 -2,-2h-6h-8.74228e-08c-1.10457,4.82823e-08 -2,0.895431 -2,2c0,0 0,7.10543e-15 0,7.10543e-15v12v0c0,1.10457 -0.895431,2 -2,2h-9h-8.74228e-08c-1.10457,-4.82823e-08 -2,-0.895431 -2,-2c0,0 0,-1.42109e-14 0,-1.42109e-14v-18.0131h-3.99l-7.28213e-08,-3.70051e-11c-1.10496,-0.000561575 -2.00024,-0.896761 -1.99968,-2.00172c0.000269422,-0.530115 0.210917,-1.03845 0.585683,-1.41338l21.994,-22.0065l-1.68542e-07,1.68624e-07c0.780468,-0.780848 2.04616,-0.781156 2.82701,-0.000687509c0.000229113,0.000229002 0.000458171,0.000458059 0.000687172,0.000687172l21.994,22.0065l4.69126e-08,4.69217e-08c0.781207,0.781359 0.781084,2.04807 -0.000275155,2.82928c-0.374911,0.374838 -0.883273,0.585539 -1.41343,0.585824h-3.998v18.013l7.10543e-15,1.68524e-07c0,1.10457 -0.895431,2 -2,2c-5.62169e-08,0 -1.12307e-07,0 -1.68524e-07,-7.10543e-15Z" fill="currentColor" fill-rule="evenodd"></path></svg>
```
Comment (k9ert) [inline src/cryptoadvance/specter/templates/includes/sidebar/sidebar.jinja:19]: Can we put that to an extra-file?

Comment (k9ert) [reply src/cryptoadvance/specter/templates/includes/sidebar/sidebar.jinja:19]: Ok, let's do it at some other time. Can you create an issue for that?

**Review by k9ert — APPROVED** [review]

**Review by moneymanolis — CHANGES_REQUESTED** [review]
When you take out the selection that Spectrum is shown in the menu, you can not set this option so it won't show in the sidebar. 
Either keep this selection option or set to be shown in the sidebar after connecting successfully for the first time. 

Also see suggestion comment.

**Comment by otkstdio** [comment]
@moneymanolis we decided to not show the plugin on the sidebar at all as the paradigm for the user is to think of it as a type of connection. It can be managed by clicking into the connection details top left of the sidebar and then select settings on the connection. Instead of having two methods and two paradigms for accessing it.

I'm fixing the icon selection now with a coming commit.

**Comment by moneymanolis** [comment]
> @moneymanolis we decided to not show the plugin on the sidebar at all as the paradigm for the user is to think of it as a type of connection. It can be managed by clicking into the connection details top left of the sidebar and then select settings on the connection. Instead of having two methods and two paradigms for accessing it.
> 
> I'm fixing the icon selection now with a coming commit.

OK, fine by me!

**Comment by moneymanolis** [comment]
I can still hardly see the favicon (too small) and I think the ghost should be white:

grafik

**Review by moneymanolis — APPROVED** [review]

**Pull request merged**